Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -217,6 +218,12 @@ template: enum class WebCore::ServiceWorkerRegistrationIdentifierType
     [Validator='WTF::ObjectIdentifierGenericBase<uint64_t>::isValidIdentifier(*toUInt64)'] uint64_t toUInt64()
 }
 
+header: <wtf/ObjectIdentifier.h>
+template: enum class WebKit::LogStreamIdentifierType
+[WebKitPlatform, CustomHeader, AdditionalEncoder=StreamConnectionEncoder] class WTF::AtomicObjectIdentifier {
```

Comment:
There is already AtomicObjectIdentifier in this file, why are you adding another one? Just add `template: enum class WebKit::LogStreamIdentifierType` to the existing one.